walletnode/api/gen/http/artworks/client/client.go

```diff
@@ -248,9 +246,7 @@ func (c *Client) ArtSearch() goa.Endpoint {
 			return nil, err
 		}
 		var cancel context.CancelFunc
-		_, cancel = context.WithCancel(ctx)
-		defer cancel()
-
+		ctx, cancel = context.WithCancel(ctx)
```

Comment:
same above:
this is bug of GOA - with this generated code - it call deferred function (cancel()) immediately and stop request immediately - while we want to listen the change in task state. please revert this back to :
```
_, cancel = context.WithCancel(ctx)
defer cancel()
````